.github/workflows/container-image.yml

```diff
@@ -1,57 +1,102 @@
-on:
-  push:
-    branches:
-    - main
-    tags:
-    - 'v*'
-
+on: push
 name: build container image
-
 jobs:
-  build:
+  build-x86_64:
+    name: Build x86_64 binary
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+      - run: rustup target add x86_64-unknown-linux-musl
+      - name: Setup musl for x86_64
+        run: |
+          curl https://musl.cc/x86_64-linux-musl-cross.tgz | tar -xz
+          echo "$PWD/x86_64-linux-musl-cross/bin" >> $GITHUB_PATH
+      - run: touch policy-server-x86_64
+      # - name: Build policy-server
+      #   env:
+      #     CC: x86_64-linux-musl-gcc
+      #   run: |
+      #     cargo build --target=x86_64-unknown-linux-musl --release
+      #     mv target/x86_64-unknown-linux-musl/release/policy-server policy-server-x86_64
```

Comment:
This smells like some debugging going on :)

Reply:
Yes, I am still checking that the image can be built properly on my fork. I will ping you here when the build has succeeded in Github actions in my fork.